src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -1292,7 +1292,7 @@ export default class Aircraft {
         const routeModel = new RouteModel(data[0]);
         const airport = window.airportController.airport_get();
         const { name: starName } = airport.starCollection.findRouteByIcao(routeModel.procedure);
-
+        let flag = airport.wip ? ' &#9983 ' : '';
```

Comment:
this can actually be removed.  what we are trying to avoid is having the flag present within the readbacks, which is what we were getting before when the flag was present in the `airport.name`.